docs/source/contributing/developing-core.md

```diff
@@ -54,22 +54,26 @@ For more information about pnpm workspaces, read the [documentation of pnpm work
 ├─ ...
 ```
 
-## Development requirements
+## Development pre-requisites
 
-To set up a Volto core development environment, you need Node.js installed in your system.
-Then, set up the package manager pnpm.
+To set up a Volto core development environment, your system must satisfy the following pre-requisites.
```

Comment:
It's great to be including content so we're consistent with https://6.docs.plone.org/install/install-from-packages.html

Would it be helpful to include a brief summary here about how the process is different from installing Plone from a release? The key differences are:
1. You have to use pnpm, not yarn or another Node package manager
2. You don't need to install Python, if you run the backend in Docker
3. You don't need to create a project using cookiecutter

Reply:
I think I added some language to make this distinction, but it can be revised. I don't want to get too deep in the details, as the details can change and once you go down a path—project versus core—you are focused on doing just that.